import traceback 
try: 
   import utils 
   print(utils) 
except Exception: 
   traceback.print_exc() 
